client/scss/components/_segmented_control.scss

```diff
@@ -0,0 +1,71 @@
+.segmented-control__drawer {
+  display: none;
+
+  .enhanced & {
+    display: block;
+
+    @include respond-to('medium') {
+      display: none;
+    }
+  }
+}
+
+.segmented-control__button-text {
+  .segmented-control__drawer.is-active & {
+    display: none;
+  }
+}
+
+.segmented-control__close {
+  display: none;
+
+  .segmented-control__drawer.is-active & {
```

Comment:
Modifiers or abstracted classes? One of the points of BEM is to avoid specificity.

Reply:
We've settled on this pattern previously (and I'm in favour of continuing to use it).